nat-lab/tests/telio.py

```diff
@@ -986,7 +974,7 @@ async def get_network_info(self) -> str:
         return ""
 
     async def check_logs_for_errors(self) -> None:
```

Comment:
Since both `check_logs_for_errors` and `save_logs` now depend on the fact that they are called only in this file and after the call to `flush_logs` I think it would be good to add something to make sure we don't accidentally call them in the wrong context. Maybe at least some sort of prefix could be added to the names, like `_` (or `_unsafe_`) to indicate those are implementation details not meant to be called from outside. Also a docstring explaining they don't flush would be good.

Reply:
Updated